InterfaceStubGenerator.Shared/InterfaceStubGenerator.cs

```diff
@@ -95,7 +95,7 @@ public void Execute(GeneratorExecutionContext context)
             var methodSymbols = new List<IMethodSymbol>();
             foreach (var group in candidateMethods.GroupBy(m => m.SyntaxTree))
             {
-                var model = compilation.GetSemanticModel(group.Key);             
```

Comment:
It's not a big deal with this PR but try not to make a bunch of whitespace-only changes, I know it can be a pain in the ass